data/campaigns/Heir_To_The_Throne/overworld/00c_northlands.cfg

```diff
@@ -0,0 +1,1573 @@
+                [/option]
+                [option]
+                    message=_"100 gold; magical training: <b>Konrad gains +2mp</b>"
+                    [command]
+                        {TRAINING_MONTAGE 100
+                            _"After rigorous physical and magical training with the saurians of the Swamp of Dread, Konrad has gained <b>+2mp</b>."
+                            ({EFFECT movement increase=2})
+                        }
+                    [/command]
+                [/option]
+                [option]
+                    # this is expensive, but is a great way for Konrad (with the Sceptre) to assassinate Asheviere in the final scenario
+                    message=_"150 gold; legendary training: <b>Konrad gains +2mp and <i>skirmisher</i></b>"
+                    [command]
+                        {TRAINING_MONTAGE 150
+                            _"After rigorous physical and magical training with the saurians of the Swamp of Dread, Konrad has gained <b>+2mp</b> and the <b><i>skirmisher</i></b> ability."
+                            ({EFFECT movement increase=2} {EFFECT new_ability (
+                                [abilities]
+                                    {ABILITY_SKIRMISHER}
+                                [/abilities]
+                            )})
+                        }
+                    [/command]
+                [/option]
+                [option]
+                    message=_"Decline"
+                    [command]
+                    [/command]
+                [/option]
+            [/message]
+        )} {/IF}
+    )}
+    #------------------------
+    # ALREADY TRAINED
+    #------------------------
+    {ELSE(
+        [message]
+            speaker=Tirix
+            message=_"Good, good, much better. Not so ssslow any more! I have no time or interessst in more training."
+        [/message]
+    )} {/IF}
+[/event]
+
+#define ATTEPT_TO_EXPLAIN_FALL_OF_KNALGA KONRAD_OR_NOBODY
+    {IF} {VARIABLE_CONDITIONAL bm_fall_of_knalga_explained not_equals yes} {THEN(
+        #------------------------
+        # HAVE ULFDAIN
+        #------------------------
+        {IF} {HAVE_UNIT( side,search_recall_list=1,yes {AND id=Ulfdain} )} {THEN(
+            [cancel_action]
+            [/cancel_action]
+            {VARIABLE bm_fall_of_knalga_explained yes}
+            {MOVE_UNDER_KONRAD( {RECALL_XY Ulfdain $stored_konrad.x $stored_konrad.y} {MOVE_UNIT id=Ulfdain "$($stored_konrad.x-1)" $stored_konrad.y} )}
+            [message]
+                speaker={KONRAD_OR_NOBODY}
+                message=_"What happened here was such a tragedy... do you know anything more than I, Ulfdain?"
+            [/message]
+            [message]
+                speaker=Ulfdain
+                message=_"Aye... ’twas a time of doom an’ great deeds, o’ fire an’ blood an’ slaughter. My father an’ his father alike perished defendin’ th’ west gate, an’ soon after fell the Doors, an’ our king Thunedain with them."
```

Comment:
As mentioned elsewhere, I think someone needs to review all the instances of these kinds of titles, but I consider the alternatives:
* 'our king, Thunedain, with them' - the lower-case is applicable as if he did not include the name.
* 'our King Thunedain with them' - Thunedain is the subject and his title is King.